test/Microsoft.AspNet.Mvc.FunctionalTests/Logging/LoggingExtensions.cs

```diff
@@ -0,0 +1,166 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using LoggingWebSite;
+using Microsoft.AspNet.WebUtilities;
+
+namespace Microsoft.AspNet.Mvc.FunctionalTests
+{
+    public static class LoggingExtensions
+    {
+        public const string RequestTraceIdQueryKey = "RequestTraceId";
+
+        /// <summary>
+        /// Gets a scope node with the given name
+        /// </summary>
+        /// <param name="activities"></param>
+        /// <param name="scopeName"></param>
+        /// <returns>A scope node if found, else null</returns>
```

Comment:
there's no need to add doc headers in test code. You're not wrong to do it, but you don't need to.